crates/core/src/types/token.rs

```diff
@@ -584,12 +590,12 @@ impl PartialOrd for DenominatedAmount {
             let ord = self.amount.raw.partial_cmp(&div_ceil);
             if let Some(Ordering::Equal) = ord {
                 if rem.is_zero() {
-                    Some(Ordering::Equal)
+                    Ordering::Equal
                 } else {
-                    Some(Ordering::Greater)
+                    Ordering::Greater
                 }
             } else {
-                ord
+                ord.unwrap()
```

Comment:
this doesn't seem right - `ord.is_none()`, no?

Reply:
could be*

Reply:
ah, the partial ord on `Uint` is always `Some`, but we can just use `cmp` in place of `partial_cmp`

Reply:
done in [2a6095f](https://github.com/anoma/namada/pull/2687/commits/2a6095f06865254e0707210ff10e032031e1ad08)

Reply:
there was a new lint about `cmp` that I had to fix a few times. I just kinda ran through it, so they may not always be the most elegant.